File: fixtures/cpp-sample/main_test.cpp
#include "greet.h"
#include <cassert>
#include <iostream>

int main() {
    assert(greet("") == "Hello, World!");
    assert(greet("C++") == "Hello, C++!");
    std::cout << "All tests passed" << std::endl;
    return 0;
}

File: fixtures/node-sample/main.test.js
const test = require("node:test");
const assert = require("node:assert/strict");

const { greet } = require("./main");

test("greet uses the default name", () => {
    assert.equal(greet(), "hello, world");
});

test("greet accepts a custom name", () => {
    assert.equal(greet("spawn"), "hello, spawn");
});
